protoc-gen-orion/testprotos/standalone_mode/example.go

```diff
@@ -0,0 +1,5 @@
+package example
+
+import "google.golang.org/grpc"
+
+var ExampleService_ServiceDesc grpc.ServiceDesc
```

Comment:
@cs-lexliu Can you variable generated from protoc-gen-go
Which version of protoc-gen-go should be used to generate it?
because current version would generate _ExampleService_ServiceDesc.

Do we have to bump the version in shared-proto scripts?

Reply:
Let me create another PR to describe how to use the tool and supported grpc version for the tool